hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java

```diff
@@ -153,6 +153,20 @@ default VALUE getReadCopy(KEY key) throws IOException {
    */
   TableIterator<KEY, ? extends KeyValue<KEY, VALUE>> iterator();
 
+  /**
+   * Returns a prefixed iterator for this metadata store.
+   * @param prefix
+   * @return
+   */
+  TableIterator<KEY, ? extends KeyValue<KEY, VALUE>> iterator(KEY prefix)
+      throws IOException;
+
+  /**
+   * Set the fixed key prefix length.
+   * @param length
+   */
+  void setFixedPrefixLength(int length);
```

Comment:
We should not set prefix length at _Table_ level.

Reply:
Sure